go/host/db/blocks_test.go

```diff
@@ -0,0 +1,89 @@
+package db
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+const magicNumber = 777
```

Comment:
Gonna guess the magic number is the minimum required for making a header ? Can a comment be added to it ?

Reply:
No, it's just a field to check is still there when the object comes back from the DB. Will rename.